pkg/sql/catalog/tabledesc/table_desc.go

```diff
@@ -74,7 +74,7 @@ func (desc *wrapper) GetPostDeserializationChanges() catalog.PostDeserialization
 func (desc *wrapper) HasConcurrentSchemaChanges() bool {
 	return (desc.DeclarativeSchemaChangerState != nil &&
 		desc.DeclarativeSchemaChangerState.JobID != catpb.InvalidJobID) ||
-		(len(desc.Mutations) > 0 && len(desc.MutationJobs) > 0)
+		len(desc.MutationJobs) > 0
```

Comment:
I actually forgot about types, but other descriptor kinds don't have mutations, so that's why tables (and types!) are special.

FYI there is no such thing as a "mutation job", just legacy schema changer jobs associated with mutations.